pkg/glusterblock/driver.go

```diff
@@ -0,0 +1,71 @@
+package glusterblock
+
+import (
+	"github.com/gluster/gluster-csi-driver/pkg/glusterblock/config"
+
+	"github.com/gluster/glusterd2/pkg/restclient"
+	"github.com/golang/glog"
+	"github.com/kubernetes-csi/drivers/pkg/csi-common"
+)
+
+const (
+	glusterBlockCSIDriverName    = "org.gluster.glusterblock"
+	glusterBlockCSIDriverVersion = "0.0.9"
+)
+
+// GfDriver is the struct embedding information about the connection to gluster cluster and configuration of CSI driver.
+type GfDriver struct {
+	client *restclient.Client
+	*config.Config
+}
+
+// New returns CSI driver
+func New(config *config.Config) *GfDriver {
```

Comment:
I think this also can  be a common function between both drivers

Reply:
I will do code refactor in separate patch